boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.dts

```diff
@@ -25,6 +25,10 @@
 		zephyr,console = &flexcomm4_lpuart4;
 		zephyr,shell-uart = &flexcomm4_lpuart4;
 	};
+
+	aliases {
+		watchdog0 = &wwdt0;
```

Comment:
put this alias in the common file I think so that the cpu1 can know about it (not sure though, @mmahadevan108 can you input here what is the strategy with these files)

Reply:
or maybe watchdog0 for cpu1 will be wwdt1 once that is supported?

Reply:
@EmilioCBen having the alias inside the cpu0 might be better since we have 2 watchdogs and wwd1 can be reserved for cpu1